components/Countries.tsx

```diff
@@ -14,7 +14,7 @@ type Props = {
 }
 
 const Countries: FunctionComponent<Props> = ({ items, searchBy }) => {
-  const countries = items.map(({ image, defaultLocale, code }, key) => {
+  const countries = items.map(({ image, defaultLocale, code }, key) => {  
```

Comment:
no big deal, but try to avoid committing these files that don't really change, it's just noise in the pull request